classes/class-loader.php

```diff
@@ -346,6 +346,8 @@ public function enqueue_public_assets() {
 			],
 			$css_blocks_creation
 		);
+
+		wp_enqueue_script( 'counter', P4GBKS_PLUGIN_URL . 'public/js/counter.js', [ 'jquery' ], '0.1', true );
```

Comment:
Could you enqueue this js only for counter block. [eg](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/develop/classes/blocks/class-cookies.php#L106)